_haikus/enligth.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Data knowledge
+author: cruz
+---
+
+please serve the content<br>
+data will spread knowledge<br>
+gave birth to light above<br>%
```

Comment:
This line is one syllable too long and has an extra character, you can try:

```
birthing light above <br>
```

Reply:
Once again thank you so much for your time. You are a hero!

Reply:
I have already submit the corrections. Hopefully this time will pass the texts :D